### DISCUSSION TITLE
Cannot view the API docs

### DISCUSSION THREAD

**Discussion by EdLeafe** [Screenly/Anthias#1828] [Q&A] [answered]
I would to be able to programmatically update the assets on my Anthias installation via an API.

The docs/README.md file says "To get started, open your browser and go to http://<ip-address>:8080/api/docs/. You should see the Swagger API docs for the endpoints." I've tried that, but I cannot connect to that port. If I ssh to the system running Anthias, I don't find anything listening on port 8080. 

How can I find out how to use the API?

**Comment by vpetersson** [comment, marked as answer]
Ah so that seems to be a bug in the rendering (ping @nicomiguelino). 

In any case, that's easy to work around. Just browse to the web interface and add /api/docs to the end and you should be good.

**Reply by EdLeafe** [reply]
Thanks! That worked.

**Reply by nicomiguelino** [reply]
Alternatively, you can click the API link on the footer part of the web UI.
Glad you noticed. Thanks!
